apps/prairielearn/src/schemas/schemas/infoQuestion.json

```diff
@@ -264,7 +264,7 @@
       "description": "Whether this question is publicly shared.",
       "type": "boolean"
     },
-    "sharedPubliclyWithSource": {
+    "sharedSourcePublicly": {
```

Comment:
This should be a safe change to make, since we have not documented or publicized this property in any way up to this point. 

As for values already in the database, there was no way to actually flip this bit in the database except manually. 

Was it done for any of the OER courses? If not the whole column is probably just still false. Anyway, the rename of the db column should be perfectly safe as well.